sc.it.homework.Concurrency/src/main/java/Concurrency/FestivalStatisticsThread.java

```diff
@@ -0,0 +1,36 @@
+package Concurrency;
+import java.util.Collections;
+import java.util.LinkedList;
+public class FestivalStatisticsThread implements Runnable {
+    String threadName;
+    static FestivalGate gate = new FestivalGate();
+    LinkedList list;
+    int fullTickets = 0;
+    int freePass = 0;
+    int vipPass = 0;
+    int oneDayPass = 0;
+    int oneDayVIP = 0;
+    public FestivalStatisticsThread(String name) {
+        threadName = name;
+        list = gate.generateAttendeeThread();
+    }
+    public void run(){

```

Comment:
this doesn't work as expected, it should wake up every 5 seconds when people are joining the festival, in your case it is called at the end